src/app/account-transfers/create-standing-instructions/create-standing-instructions.component.ts

```diff
@@ -169,8 +173,8 @@ export class CreateStandingInstructionsComponent implements OnInit {
           'toOfficeId': this.officeId,
           'toClientId': this.clientId
         });
-        this.createStandingInstructionsForm.controls['toOfficeId'].disable();
-        this.createStandingInstructionsForm.controls['toClientId'].disable();
+        this.ToOfficeId = true;
+        this.ToClientId = true;
```

Comment:
So it is disabled anyway?

Reply:
Actually, disable() method, it not only prevents the user from interacting with form but also excludes it from the form values when the form is submitted.

Reply:
The request going to the backend did not include the values for 'ToOfficeID' and 'toClientId'

Reply:
i dont really understand how it is doing this... :/

Reply:
@adamsaghy are you talking about the functionality or you understand the functional requirement but you don't understand how the code changes made are meeting the requirement?

Reply:
the latter

Reply:
Actually @adamsaghy we didn't want to exclude the toOfficeId and toClientId, we just wanted to grey out those filling in the form.
disable() method in .ts grey. out the filling and also exclude those keys from the form, that's why I removed the .diable() method

Reply:
so it is always greyed out?

if i understand correctly at init it is marked as disabled and i dont see any logic which would change it.

Reply:
No not always greyed out, it is only gryed out after we select destination as it own account

Reply:
https://github.com/openMF/web-app/assets/76156941/b439ff74-0d9c-405a-8e89-39c9bee36648